metadata-ingestion/src/datahub/ingestion/source/sql/teradata.py

```diff
@@ -631,13 +634,14 @@ def __init__(self, config: TeradataConfig, ctx: PipelineContext):
                 ),
             )
 
-            setattr(  # noqa: B010
-                TeradataDialect,
-                "get_view_definition",
-                lambda self, connection, view_name, schema=None, **kw: optimized_get_view_definition(
-                    self, connection, view_name, schema, tables_cache=tables_cache, **kw
-                ),
-            )
+            # Disabling the below because the cached view definition is not the view definition the column in tablesv actually holds the last statement executed against the object... not necessarily the view definition
```

Comment:
This was the most inefficient part right? Because we had to make ~4 queries per view. Are you saying our old fix doesn't work properly and there's no good path forward

Reply:
Unfortunately, there is no better way to do it on Teradata.  :(
I think this shouldn't be as bad as it was before, as it still runs way fewer queries than before.